src/components/TokenField/index.tsx

```diff
@@ -13,8 +13,8 @@ interface CustomProps {
   approxUSD: string;
 }
 
-type Ref = HTMLInputElement;
-const TokenField = React.forwardRef<Ref, Props>(
+// TODO: replace its use cases with `src\componentLibrary\TokenField`
```

Comment:
This isn't quite right - we'll be keeping these components for as long as we maintain the existing UI (which could be a while), phasing it out as and when Jay redesigns the features/components (i.e. there won't ever be a task to go through and swap the components round).

What would probably be more useful is a not indicating that this is a legacy component and shouldn't be used in new features.

Reply:
I got you.
I assumed that Jay might replicate some of the existing UIs. This kind of component is necessary for almost all of DeFi projects in my experience. But again I got your points. Then I will remove the TODO.

Reply:
Thanks :)

Reply:
Done